src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -264,7 +264,7 @@ def acr_queue(cmd,
                 client_registries, registry_name, resource_group_name, source_location, tar_file_path, docker_file_path)
         else:
             raise CLIError(
-                "'--source-location' should be a local directory path or remote url.")
+                "'--context -c' should be a local directory path or remote url.")
```

Comment:
This kind of validation should go to `_params.py` if possible so it's easier to track and also the validation could be done before spending time on the previous steps.

Reply:
@djyou does doing stuff like `os.exists` make sense in params.py? If so please make a new issue to keep track of it